exercises/kindergarten-garden/src/test/kotlin/KindergartenGardenTest.kt

```diff
@@ -0,0 +1,87 @@
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class KindergartenGardenTest {
+
+    @Test
+    fun singleStudent() {
+        val student = "Alice"
+        val plants = "RC\nGG"
+        val expected = listOf(Plant.RADISHES, Plant.CLOVER, Plant.GRASS, Plant.GRASS)
+
+        assertEquals(expected, KindergartenGarden(plants).getPlantsOfStudent(student))
+    }
+
+    @Test
```

Comment:
Please @Ignore all tests except the first.

This gives users a chance to build up their solutions test-by-test.